src/retinanet.py

```diff
@@ -233,7 +235,7 @@ def forward(self, output, target):
         return loss
 
 
-class DataEncoder(object):
+class BaseDataHandler():
     def __init__(self):
         self.anchor_areas = [32*32., 64*64., 128*128., 256*256., 512*512.]  # p3 -> p7
```

Comment:
@apyskir @taraspiotr it feel like both this and aspect ratios should be specified in some sort of the config since they look like magic numbers here. On the other hand keeping it close to the model may make sense. 
What are your thoughts?

Reply:
@taraspiotr Do we anticipate some sort of changing these things? I am not sure, so maybe they can stay here. Unlike input image shape, that has to be parametrized.